target/linux/ramips/dts/mt7621_dlink_dir-xx60-a1.dtsi

```diff
@@ -47,12 +47,14 @@
 		};
 
 		led_net_orange: net_orange {
-			label = "orange:net";
+			function = LED_FUNCTION_WAN;
+			color = <LED_COLOR_ID_ORANGE>;
 			gpios = <&gpio 4 GPIO_ACTIVE_LOW>;
 		};
 
 		net_white {
-			label = "white:net";
+			function = LED_FUNCTION_WAN;
+			color = <LED_COLOR_ID_WHITE>;
 			gpios = <&gpio 3 GPIO_ACTIVE_LOW>;
```

Comment:
This changes the name of the LED. The name is referenced here: https://github.com/openwrt/openwrt/blob/main/target/linux/ramips/mt7621/base-files/etc/board.d/01_leds#L97